configure.ac

```diff
@@ -251,29 +251,21 @@ AC_TYPE_INT16_T
 AC_TYPE_INT32_T
 AC_TYPE_INT64_T
 
+PKG_CHECK_MODULES([libqb], [libqb])
+PKG_CHECK_VAR([libqb_PREFIX], [libqb], [prefix])
+
 if test "x$enable_man" = "xyes"; then
 	AC_ARG_VAR([DOXYGEN], [override doxygen executable])
 	AC_CHECK_PROGS([DOXYGEN], [doxygen], [no])
 	if test "x$DOXYGEN" = xno; then
 		AC_MSG_ERROR(["Doxygen command not found"])
 	fi
-	# required by doxyxml to build man pages dynamically
-	# Don't let AC_PROC_CC (invoked by AX_PROG_CC_FOR_BUILD) replace
-	# undefined CFLAGS_FOR_BUILD with -g -O2, overriding our special OPT_CFLAGS.
-	: ${CFLAGS_FOR_BUILD=""}
-	AX_PROG_CC_FOR_BUILD
-	saved_PKG_CONFIG="$PKG_CONFIG"
-	saved_ac_cv_path_PKG_CONFIG="$ac_cv_path_PKG_CONFIG"
-	unset PKG_CONFIG ac_cv_path_PKG_CONFIG
-	AC_PATH_PROG([PKG_CONFIG], [pkg-config])
```

Comment:
Hi Fabio, this is getting elegant, thank you for taking care! I'm afraid my last point about mixing the host- and build-system libqbs still stands, though. I think you can't really drop this excursion for the build-system `pkg-config`, because the `PKG_CHECK_*` macros use `AC_PATH_TOOL` to find the host-system `pkg-config`, thus `PKG_CHECK_VAR` uses that, but to find a `doxygen2man` to run during the build you need the build-system `pkg-config`, which is found by `AC_PATH_PROG` as above. (Since it's a script, it will run cross-architecture, but it'll always use the pkg-config directory of its native architecture by default). I mean something like
```
saved_PKG_CONFIG="$PKG_CONFIG"
saved_ac_cv_path_PKG_CONFIG="$ac_cv_path_PKG_CONFIG"
unset PKG_CONFIG ac_cv_path_PKG_CONFIG
AC_PATH_PROG([PKG_CONFIG], [pkg-config])
PKG_CHECK_VAR([libqb_BUILD_PREFIX], [libqb], [prefix])
PKG_CONFIG="$saved_PKG_CONFIG"
ac_cv_path_PKG_CONFIG="$saved_ac_cv_path_PKG_CONFIG"
```
leading to
```
AC_PATH_PROG([DOXYGEN2MAN], [doxygen2man], [no], [$libqb_BUILD_PREFIX/bin$PATH_SEPARATOR$PATH])
```
(Note that `$PATH_SEPARATOR` is build-system specific as well.)
Hope it makes sense, I can't actually test it right now, I'm rather overwhelmed by urgent work for the government.
And most of all I hope you're all right and safe.

Reply:
@wferi thanks for taking the time to review. I understand now what you mean!
I´ll try to clean it up next week, this is hardly urgent in any way and I am sure your work is far more important.

Take care and be safe!
Fabio

PS we are all good here :)